user_guide_src/source/changelogs/v4.6.4.rst

```diff
@@ -40,6 +40,7 @@ Bugs Fixed
 - **Database:** Fixed a bug in ``Database::connect()`` which was causing to store non-shared connection instances in shared cache.
 - **Database:** Fixed a bug in ``Connection::getFieldData()`` for ``SQLSRV`` and ``OCI8`` where extra characters were returned in column default values (specific to those handlers), instead of following the convention used by other drivers.
 - **Database:** Fixed a bug in ``BaseBuilder::compileOrderBy()`` where the method could overwrite ``QBOrderBy`` with a string instead of keeping it as an array, causing type errors and preventing additional ``ORDER BY`` clauses from being appended.
+- **Database:** Fixed a bug in ``SQLite3`` where an incorrect condition caused the password parameter to be ignored when it was an empty string.
```

Comment:
Should this read as ignored when it was not an empty string?

Reply:
Yes.

> Fixed a bug in `SQLite3` where the password parameter was ignored unless it was an empty string.

What do you think now? More clear?

Reply:
Yes

Reply:
Updated - thank you.